rust/utils/run-locally/src/main.rs

```diff
@@ -171,6 +171,22 @@ fn main() -> ExitCode {
         .hyp_env("CHAINS_SEALEVELTEST2_SIGNER_KEY", RELAYER_KEYS[4])
         .hyp_env("RELAYCHAINS", "invalidchain,otherinvalid")
         .hyp_env("ALLOWLOCALCHECKPOINTSYNCERS", "true")
+        .hyp_env(
+            "GASPAYMENTENFORCEMENT",
+            r#"[{
+                "type": "minimum",
+                "payment": "1",
+                "matchingList": [
+                    {
+                        "originDomain": ["13375","13376"],
```

Comment:
We might actually want to enforce the gas payment for all chains. We currently do a check for if the gas payment count is as expected, but it would test some extra logic to add this to at least 1 of the test EVM chains.

Also you should be able to write the numbers without quotes

Reply:
I'm not sure which domain to set for the EVM ones actually. Is it always `31337` for anvil and could I use this one?

Reply:
just to keep scope small i'm comfortable leaving this till later